README.md

```diff
@@ -48,7 +48,7 @@ most recent version executing the `symfony self-update` command.
 Usage
 -----
 
-If you have PHP 5.4 or higher, there is no need to configure a virtual host
+If you have PHP `5.4` or higher, there is no need to configure a virtual host
```

Comment:
We can reword this now that we are on Symfony 3. People will actually already need to have PHP 5.5.9 or higher (as you changed above).